client/src/packets.rs

```diff
@@ -0,0 +1,47 @@
+use rtpeeker_common::packet::Packet;
+use std::cell::RefCell;
+use std::collections::{btree_map::Values, BTreeMap};
+use std::rc::Rc;
+
+pub type RefPackets = Rc<RefCell<Packets>>;
+
+#[derive(Debug, Default)]
+pub struct Packets {
+    packets: BTreeMap<usize, Packet>,
+}
+
+impl Packets {
+    pub fn get(&self, id: usize) -> Option<&Packet> {
+        self.packets.get(&id)
+    }
```

Comment:
This function is probably redundant. What do you think to remove it?

Reply:
The `packets` field of `Packets` is not public, as it should. All of these functions that you marked as redundant make up the API to `Packets` and are used elsewhere.